playbook-examples/manage_cnp.yml

```diff
@@ -0,0 +1,106 @@
+# Original work Copyright 2019-2020, EnterpriseDB Corporation
+#
+# Installs Roles and Collections requirements
+# ansible-galaxy install --force -r requirements.yml
+#
+# pip install openshift pyyaml kubernetes
+# pip install pyhelm
+---
+- name: Manage a CloudNativePG or CloudNative Postgres Cluster Playbook
+  hosts: localhost
+  gather_facts: false
+
+  vars:
+    # Options"
+    # add-db, add-table, add-index, add-view
+    # drop-db, drop-table, drop-index, drop-view
+    - cnp_task: "drop-db"
```

Comment:
Is there some way to allow more than one value for the `cnp_task` variable? I think the ability to run the playbook once and perform multiple actions would be necessary for ease-of-use.

Reply:
Added functionality for lists